application/src/Weekend.php

```diff
@@ -4,49 +4,81 @@
 
 class Weekend
 {
+    protected function getMapping()
+    {
+        $mapping = [
```

Comment:
this map could be a const instead of this method

const MAPPING => [...];

It would then be used with `self::MAPPING` instead of `$this->getMapping()`

Reply:
Done in c85d9b77c7293c186477cb39d7d367c4ffeff31b.
Does that look good to you?

Reply:
Absolutely perfect! Well done :ok_hand: